.github/workflows/publish-iceberg-rest-fixture-docker.yml

```diff
@@ -28,6 +28,7 @@ on:
 env:
   DOCKER_IMAGE_TAG: iceberg-rest-fixture
   DOCKER_IMAGE_VERSION: latest
+  DOCKER_REPOSITORY: apache
```

Comment:
```suggestion
  DOCKER_REPOSITORY: apache
  DOCKER_IMAGE_TAG: iceberg-rest-fixture
  DOCKER_IMAGE_VERSION: latest
```
nit

Reply:
keys were alphabetically sorted before. I didn't get the intention behind this comment.

Reply:
this was to match the order of the docker name 

part of:
```
docker push $DOCKER_REPOSITORY/$DOCKER_IMAGE_TAG:$DOCKER_IMAGE_VERSION
```

again, nit comment :)